### PR TITLE
wallet: Update wallet utxo set when new keys are found for already known transactions

### PR DESCRIPTION
**Pull request by UdjinM6**

### CODE REVIEW

**Review by xdustinface — DISMISSED** [review]
utACK

File: src/wallet/wallet.cpp [hunk 1]
```diff
@@ -1149,6 +1149,17 @@ bool CWallet::AddToWallet(const CWalletTx& wtxIn, bool fFlushOnClose)
             wtx.fFromMe = wtxIn.fFromMe;
             fUpdated = true;
         }
+
+        auto mnList = deterministicMNManager->GetListAtChainTip();
+        for(unsigned int i = 0; i < wtx.tx->vout.size(); ++i) {
```
Comment (xdustinface) [inline src/wallet/wallet.cpp:1154]:
```suggestion
        for (unsigned int i = 0; i < wtx.tx->vout.size(); ++i) {
```
Just saw this the moment i pressed for approval 🙈

**Review by xdustinface — APPROVED** [review]
utACK

**Review by PastaPastaPasta — APPROVED** [review]
utACK

**Pull request merged**